### PR TITLE
added imagePullSecrets

### PR DESCRIPTION
**Pull request by abisare**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://developercertificate.org) signed
- [ ] Chart Version bumped (if the pr is an update to an existing chart)
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

File: charts/graphite/templates/statefulset.yaml [hunk 1]
```diff
@@ -92,6 +92,10 @@ spec:
             claimName: {{ if .Values.persistence.existingClaim }}{{ .Values.persistence.existingClaim }}{{- else }}{{ template "graphite.fullname" . }}-pvc{{- end }}
 {{- else }}
           emptyDir: {}
+{{- end }}
+{{- if .Values.pullSecret }}
```
Comment (monotek) [inline charts/graphite/templates/statefulset.yaml:96]: please add it like:
```
      {{- with .Values.imagePullSecrets }}
      imagePullSecrets:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```

File: charts/graphite/values.yaml [hunk 2]
```diff
@@ -5,6 +5,8 @@ image:
   tag: 1.1.7-6
   pullPolicy: IfNotPresent
 
+#pullSecret: some-deploy-secret
```
Comment (monotek) [inline charts/graphite/values.yaml:8]: please add it like:
```
imagePullSecrets: ""
```

**Review by monotek — CHANGES_REQUESTED** [review]
please raise minor chart version and sign your commits.

**Comment by rpahli** [comment]
pls also increase the chart version

**Comment by abisare** [comment]
sorry for late response. fixed

**Review by monotek — CHANGES_REQUESTED** [review]
New feature should raise minor version

**Comment by rpahli** [comment]
@abisare can you pls update the minor version and also sing you commits then I can merge the PR.

**Pull request closed without merging**